test/metamorpho_tests.tree

```diff
@@ -262,13 +262,13 @@
         │       └── revert with SupplyCapExceeded(id)
         ├── when totalWithdrawn > totalSupplied
         │   └── it should add totalWithdrawn - totalSupplied to idle
-        └── when totalWithdrawn <= totalSupplied
-            └── it should compute idleSupplied = totalSupplied - totalWithdrawn
-                ├── when idle < idleSupplied
-                │   └── revert with InsufficientIdle()
-                └── when idle >= idleSupplied
-                    └── it should remove idleSupplied from idle
-
+        ├── when totalWithdrawn <= totalSupplied
+        │   └── it should compute idleSupplied = totalSupplied - totalWithdrawn
+        │       ├── when idle < idleSupplied
+        │       │   └── revert with InsufficientIdle()
+        │       └── when idle >= idleSupplied
+        │           └── it should remove idleSupplied from idle
+        └── it should emit ReallocateIdle(idle)
```

Comment:
it should also emit other market-dependent events!

Reply:
f**k